test/unit/spec/EnrollU2F_spec.js

```diff
@@ -65,10 +67,19 @@ function (Okta,
       });
     }
 
+    function mockChrome(isAvailable){
+      BrowserFeatures.isChrome = function () {};
```

Comment:
Since we are spying on the function in the next line, I don't think we need to override it (function already exists in BrowserFeatures)
If I remember correctly, for ```winow.u2f``` I needed to do that since the function didn't exist